### PR TITLE
SUBMARINE-217. Add Submarine Workbench introduction document

### PR DESCRIPTION
**Pull request by liuxunorg**
### What is this PR for?
Submarine workbench introduction document mainly contains following parts:
1. workbench homepage
2. Project create/list/delete/modify manager

### What type of PR is it?
[Documentation]

### Todos
* [ ] - Interpreter create/list/delete/modify manager
* [ ] - Job create/list/delete/modify manager
* [ ] - Data create/list/delete/modify manager
* [ ] - Model create/list/delete/modify manager
* [ ] - System user/role/team/organization manager

### What is the Jira issue?
* https://issues.apache.org/jira/browse/SUBMARINE-217

### How should this be tested?
* [CI Pass](https://travis-ci.org/liuxunorg/hadoop-submarine/builds/594576189)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? Yes

### CODE REVIEW

**Comment by pingsutw** [comment]
Thanks @liuxunorg for the great work.
There are some typo, frok should be fork

File: docs/workbench/README.md [hunk 1]
```diff
@@ -12,65 +12,164 @@
...
+
+In the Project page, all the projects created by the user themselves are displayed as cards.
+
+![image-20191007161424534](assets/workspace-project.png)
+
+Each Project card consists of the following sections:
+
+1. **Project Type**：Submarine currently supports six types of machine learning algorithm frameworks and development languages: `Notebook`, `Python`, `R`, `Scala`, `Tensorflow`, and `PyTorch`, which are identified by corresponding icons in the project card.
+2. **Project Tags**：Users can tag each Project with different tags for easy searching and management.
+3. **Github/Gitlab integrated**：Submarine Workbench is system integrated with `Github`/`Gitlab`, and each Project can perform `Watch`, `Star`, `Frok`, and `Comment `operations in Workbench.
```
Comment (jojochuang) [inline docs/workbench/README.md]: Frok --> Fork?

File: docs/workbench/README.md [hunk 2]
```diff
@@ -12,65 +12,164 @@
...
+# Submarine Workbench Introduction
+
+Submarine Workbench is a WEB system for data scientists.
+Data scientists can interactively access notebooks, submit/manage jobs, manage models, create model training workflows, access data sets, and more through Submarine Workbench.
+
+## Register
+
+Everyone who needs to use Submarine for machine learning algorithm development can log in to Submarine Workbench's WEB homepage. On the homepage, click the registration link, fill in the user name, email address and password to register the user. At this time, the user status is `waiting for the audit` status. 
+
+After receiving the registration request from the user in Submarine Workbench, the administrator sets the operation authority according to the user's needs, sets the user's organization and allocates resources, and sets the user status to `pass the audit`. The user can log in to the Submarine Workbench. Different users have different Operational authority.
```
Comment (jojochuang) [inline docs/workbench/README.md]: operation authority --> permission

File: docs/workbench/HowToRun.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+`submarine-env.sh` is automatically executed each time the `workbench.sh` script is executed, so we can set the `workbench.sh` script and the environment variables in the `WorkbenchServer` process via `submarine-env.sh`.
+
+| Name                | Variable                                                     |
+| ------------------- | ------------------------------------------------------------ |
+| JAVA_HOME           | Set your java home path, default is `java`.                  |
+| WORKBENCH_JAVA_OPTS | Set the JAVA OPTS parameter when the Workbench process starts. If you need to debug the Workbench process, you can set it to `-agentlib:jdwp=transport=dt_socket, server=y,suspend=n,address=5005` |
+| WORKBENCH_MEM       | Set the java memory parameter when the Workbench process starts. |
+| MYSQL_JAR_URL       | The customized URL to download mysql jdbc jar.               |
+| MYSQL_VERSION       | The version of mysql jdbc jar would be downloaded. The default value is 5.1.39. It's used to generate the default value of MYSQL_JDBC_URL|
```
[the remark this answers is not in the document]
Comment (liuxunorg) [reply docs/workbench/HowToRun.md]: Done.

File: docs/workbench/README.md [hunk 4]
```diff
@@ -12,65 +12,164 @@
...
-
-## Compile
-
-[Build From Code Guide](./development/BuildFromCode.md)
-
-```$xslt
-cd submarine/submarine-dist/target/submarine-dist-<version>/submarine-dist-<version>/
-./bin/workbench-daemon.sh [start|stop|restart]
-```
+
```
[the remark this answers is not in the document]
Comment (liuxunorg) [reply docs/workbench/README.md]: Done.

File: docs/workbench/README.md [hunk 5]
```diff
@@ -12,65 +12,164 @@
...
+English | [简体中文](./README.zh-CN.md)
+
+# Submarine Workbench Introduction
+
+Submarine Workbench is a WEB system for data scientists.
+Data scientists can interactively access notebooks, submit/manage jobs, manage models, create model training workflows, access data sets, and more through Submarine Workbench.
+
+## Register
+
+Everyone who needs to use Submarine for machine learning algorithm development can log in to Submarine Workbench's WEB homepage. On the homepage, click the registration link, fill in the user name, email address and password to register the user. At this time, the user status is `waiting for the audit` status.
```
[the remark this answers is not in the document]
Comment (liuxunorg) [reply docs/workbench/README.md]: Done.

File: docs/workbench/README.md (hunk 2 above)
Comment (liuxunorg) [reply docs/workbench/README.md]: Done.

**Pull request closed without merging**